src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js

```diff
@@ -61,8 +61,8 @@ export const AzureComputeModalBase = ({
   const [deleteDiskSelected, setDeleteDiskSelected] = useState(false);
   const hasGpu = () => !!azureMachineTypes[computeConfig.machineType]?.hasGpu;
   // Lifecycle
-  useOnMount(
-    _.flow(
+  useOnMount(() => {
```

Comment:
`useEffect` functions must return nothing or a cleanup function. Async functions return Promises and so cannot be used as effect functions. The workaround is to create an async function and immediately call it within a synchronous effect function.